WarOfEmpires/Views/Market/Sell.cshtml

```diff
@@ -0,0 +1,164 @@
+﻿@model WarOfEmpires.Models.Markets.MarketModel
+
+@{
+    ViewBag.Title = "Sell resources";
+}
+
+<h2>Sell resources</h2>
+
+<p>
+    Send out new caravans to sell your resources on the market, or call back caravans currently on the market. You are able to send off multiple caravans at once,
+    provided you have enough available merchants.
```

Comment:
Please include information about the sales tax on this page

Reply:
Done!